dockerBuild.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+VERSION=${1:-master}
+if [ "$VERSION" == "master" ]; then
+  VERSION="latest"
+fi
+PUBLISH=${2:-false}
+
+set -e
+
+echo "---------------------"
+echo "Building vault-admin"
+echo "---------------------"
+
+docker run --rm -v "$PWD":/go/va -w /go/va golang:1.13-alpine \
+sh -c "apk add bash zip && ./build.sh ${VERSION} linux"
```

Comment:
Any reason to call `apk add bash zip` twice here?  Also, that means this script will only work on `alpine` machines (can't run this file on other flavors or macos)?